x-pack/solutions/observability/plugins/synthetics/common/rules/synthetics/translations.ts

```diff
@@ -12,7 +12,7 @@ export const SyntheticsMonitorStatusTranslations = {
     'xpack.synthetics.alerts.syntheticsMonitorStatus.defaultActionMessage',
     {
       // the extra spaces before `\n` are needed to properly convert this from markdown to an HTML email
-      defaultMessage: `Monitor "{monitorName}" is {status} from {locationNames}.{pendingLastRunAt} - Elastic Synthetics\n\nDetails:\n\n- Monitor name: {monitorName}  \n- {monitorUrlLabel}: {monitorUrl}  \n- Monitor type: {monitorType}  \n- Checked at: {checkedAt}  \n- From: {locationNames}  \n- Reason: {reason} \n- Error received: {lastErrorMessage}  \n{linkMessage}`,
+      defaultMessage: `Monitor "{monitorName}" is {status} from {locationNames}.{pendingLastRunAt} - Elastic Synthetics\n\nDetails:\n\n- Monitor name: {monitorName}  \n- {monitorUrlLabel}: {monitorUrl}  \n- Monitor type: {monitorType}  \n- Checked at: {checkedAt}  \n- From: {locationNames}  \n- Reason: {reason} \n- Error received: {lastErrorMessage}  \n{stepInfo}{linkMessage}`,
```

Comment:
@yiannisnikolopoulos I haven't tested it manually yet, but maybe you need to add `\n` after {stepInfo}?

`\n{stepInfo} \n{linkMessage}`